### ISSUE TITLE
👌 IMPROVE: Use Jupyter notebooks for executing "Running calculations" section

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-tutorials#247]
We currently suggest using the `verdi shell` for section 3.4 "Creating and launching calculations", and to copy things into a script.

Copying multi-line strings from the ipython shell is very tedious (prompts, extra dots, ...), and we should probably suggest running this subsection in the jupyter notebook server instead.

https://aiida-tutorials--245.org.readthedocs.build/en/245/pages/2020_Intro_Week/sections/running.html#creating-and-launching-calculations

**Comment by chrisjsewell** [comment]
I'm all for using notebooks 👍 
Although I would note that, with my update of sphinx-copybutton (https://github.com/executablebooks/sphinx-copybutton/pull/82), the code cells copy button will now copy ipython shell lines, with prefixes removed 😁 (give it a go)

**Comment by ltalirz** [comment]
Right, but if I remember correctly, they may need to copy their own code (PKs etc. )

**Comment by mbercx** [comment]
@ramirezfranciscof we'll discuss how to execute this once you have a better idea of the content for the "Running calculations" section.
